CME/pages/CMEQuizPage.php

```diff
@@ -621,6 +627,31 @@ protected function sendCompletionEmail()
 	}
 
 	// }}}
+	// {{{ protected function addCompletionMessage()
+
+	protected function addCompletionMessage()
+	{
+		if ($this->response->isPassed()) {
+			$message = new SwatMessage(
+				sprintf(
+					CME::_('Congratulations on passing the %s quiz!'),
+					$this->getQuizTitle()
+				)
+			);
+
+			$account = $this->app->session->account;
+			if (!$account->isEvaluationComplete($this->credits->getFirst())) {
```

Comment:
We probably should match what we [here](https://github.com/silverorange/Rap/blob/master/Rap/pages/RapQuizPage.php#L119). Right now, I think, you'll get the secondary content if there you haven't completed the eval or if there isn't an eval setup.